crates/valence_protocol/src/lib.rs

```diff
@@ -396,7 +361,7 @@ mod derive_tests {
     #[allow(unconditional_recursion)]
     fn has_impls<'a, T>()
```

Comment:
type parameter goes unused in function definition

Reply:
The type param asserts that the input types have the expected bounds.